lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -60,9 +60,9 @@ function runLighthouse(url, configPath, isDebug) {
     '--port=0',
   ];
 
-  if (isDebug) {
-    args.push('-GA');
-  }
+  // Save artifacts
+  args.push('-GA');
```

Comment:
we should specify a tmp path if we're *always* doing this otherwise every smoke run will overwrite the users saved artifacts